flybywiresim/a32nx
# feat(fmgc): runway-by-itself approaches

Fixes #954

## Summary of Changes

Adds support runway-by-itself approach, which gives a CF leg on runway course 5 NM long.

Also fixes a small bug with final approaches that creates a lot of uncaught exceptions for RNAV approaches (sentry).

~~Draft as I'm awaiting confirmation of the approach naming... but could still be reviewed before that.~~
Confirmed that the 5 NM fix is simply called "CF", the approach name is just the runway number, and there is no approach message on the ND.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/9995998/161373179-77f9ec8f-c49a-4002-8a83-80940c6e5753.png)
![image](https://user-images.githubusercontent.com/9995998/161408248-b607c863-fa65-4672-b630-a434ac7326c7.png)


## References

Honeywell pilot's guide. Honeywell pilot.




## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Try adding a runway-by-itself approach (bottom of the approaches list), check that it gives a 5 NM final leg
- Try "regular" approaches to ensure they still work okay


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
